ci-operator/step-registry/shiftstack/cleanup/shiftstack-cleanup-workflow.yaml

```diff
@@ -5,5 +5,7 @@ workflow:
     - ref: openstack-conf-clouds
     test:
     - ref: shiftstack-cleanup
+    post:
+    - ref: openstack-deprovision-credentials
```

Comment:
I can't remember why password authentication is mandatory for the cleanup job... but it is, and this means that this step is not necessary